resources/mvn-exec.sh

```diff
@@ -8,4 +8,4 @@ fi
 
 . $CONFIG
 
-exec mvn compile exec:exec -Dexec.executable=java -Dexec.args="-cp %classpath ${JAVA_SYS_PROPS} org.jitsi.videobridge.Main --domain=\"${JVB_HOSTNAME}\" --host=\"${JVB_HOST}\" --port=\"${JVB_PORT}\" --secret=\"${JVB_SECRET}\" --opts=${JVB_OPTS}"
+exec mvn compile exec:exec -Dexec.executable=java -Dexec.args="-cp %classpath ${JAVA_SYS_PROPS} org.jitsi.videobridge.Main --opts=${JVB_OPTS}"
```

Comment:
Should `--opts=${JVB_OPTS}` just be `${JVB_OPTS}` or am I missing something? I don't see a `--opts` defined anywhere.

Reply:
Yes, I think so, good catch

Reply:
I added a commit to fix that.